java/src/test/java/ai/rapids/cudf/BinaryOpTest.java

```diff
@@ -1098,4 +1099,17 @@ public void testLogBase2() {
       assertColumnsAreEqual(expected, answer, "log2");
     }
   }
+
+  @Test
+  public void testArctan2() {
+    Double[] xValues = TestUtils.getDoubles(20342309423423L, 10, false);
```

Comment:
is 10 values enough to actually get NaN, Infinity, and -Infinity?

Reply:
I have increased the values to 50 regardless.